unit_test.sh

```diff
@@ -9,6 +9,13 @@ set -o pipefail
 echo "Running linter..."
 golangci-lint run
 
+timestamp=`date +%Y-%m-%d_%H-%M-%S`
+mkdir -p test_results/${timestamp}
+mkdir -p test_results/latest
+
+echo "Running gosec..."
+gosec -fmt=junit-xml -exclude=G104 -out=test_results/latest/gosec_results.xml ./...
```

Comment:
For rules we are excluding in the command line invocation... should we have follow-up stories to address them?

Reply:
this one is BCDA-554 as described in the PR.

Reply:
Got it!